docs/how-to/reactive.md

```diff
@@ -0,0 +1,112 @@
+# How to set up reactive spawning
+
+The charm provides an experimental feature to spawn runners reactively, depending on the jobs requested by GitHub.
+This feature is disabled by default and can be enabled by integrating the charm with a MongoDB database.
+
+## Requirements
+
+You need to deploy a webhook router, which listens for incoming jobs from GitHub, transforms them into
+labels and stores those labels in a MongoDB database. You can use the [GitHub Runner Webhook Router](https://charmhub.io/github-runner-webhook-router) for this purpose.
+The router and the GitHub runner charm must both be integrated with the same mongodb database.
+
+
+## Steps
+We are going to showcase the steps required to set up a reactive spawning environment with three runner flavors (large, large-arm, small) and a MongoDB database as a message queue.
+
+Note, that the specific revisions/channels in the steps are only marked here to have the howto reproducible, you should adapt these to your needs.
```

Comment:
Nitpick: "to have the howto reproducible" --> "for reproducibility"

Reply:
Another nitpick: "you should adapt these"...these what? These steps? It would be good to clarify